tests/surveys/complex-survey/configuration.json

```diff
@@ -0,0 +1,89 @@
+{
+    "admin_name": "fastsurvey",
+    "survey_name": "complex-survey",
+    "title": "Complex Survey Test",
+    "description": "",
+    "start": 1000000000,
+    "end": 2000000000,
+    "mode": 1,
+    "fields": [
+        {
+            "type": "Email",
+            "title": "What is your email address?",
+            "description": "",
+            "regex": ".*",
+            "hint": "Your email address ..."
+        },
+        {
+            "type": "Option",
+            "title": "I have read and agree to the terms and conditions",
+            "description": "",
+            "mandatory": true
```

Comment:
Why did you choose to use `mandatory` instead of `required`?

Reply:
**Answer:** Cerberus uses the `required` keyword.

So using the same keyword in a dict to be validated needs some workaround. This is not worth fixing right now!